Releases/v1.1.0/docker/.gitignore

```diff
@@ -0,0 +1 @@
+requirements.txt
```

Comment:
if this file is used in the docker file, it will need to be included in the repository